File: Serene/Serene.Script/Imports/FormContexts/Meeting.MeetingDecisionForm.cs

namespace Serene.Meeting
{
    using Serenity;
    using Serenity.ComponentModel;
    using System;
    using System.Collections;
    using System.Collections.Generic;
    using System.ComponentModel;
    using System.Runtime.CompilerServices;
    using Serene.Northwind;

    public partial class MeetingDecisionForm : PrefixedContext
    {
        public MeetingDecisionForm(string idPrefix) : base(idPrefix) {}
    
        public IntegerEditor MeetingId { get { return ById<IntegerEditor>("MeetingId"); } }
        public IntegerEditor ResolutionStatusId { get { return ById<IntegerEditor>("ResolutionStatusId"); } }
        public IntegerEditor DisplayOrder { get { return ById<IntegerEditor>("DisplayOrder"); } }
        public StringEditor IsActive { get { return ById<StringEditor>("IsActive"); } }
        public IntegerEditor MeetingAgendaId { get { return ById<IntegerEditor>("MeetingAgendaId"); } }
        public StringEditor Description { get { return ById<StringEditor>("Description"); } }
        public IntegerEditor ImageFileId { get { return ById<IntegerEditor>("ImageFileId"); } }
        public IntegerEditor ResponsibleContactId { get { return ById<IntegerEditor>("ResponsibleContactId"); } }
        public DateEditor DueDate { get { return ById<DateEditor>("DueDate"); } }
    }
}

File: Serene/Serene.Web/Modules/Common/Imports/ServerTypings/Meeting.MeetingDecisionForm.ts
namespace Serene.Meeting {
    export class MeetingDecisionForm extends Serenity.PrefixedContext {
        static formKey = 'Meeting.MeetingDecision';

    }

    export interface MeetingDecisionForm {
        MeetingId: Serenity.IntegerEditor;
        AgendaId: Serenity.IntegerEditor;
        Description: Serenity.StringEditor;
        DecisionNumber: Serenity.IntegerEditor;
        ResponsibleContactId: Serenity.IntegerEditor;
        DueDate: Serenity.DateEditor;
        ResolutionStatus: Serenity.IntegerEditor;
        Images: Serenity.StringEditor;
        Attachments: Serenity.StringEditor;
    }

    [['MeetingId', () => Serenity.IntegerEditor], ['AgendaId', () => Serenity.IntegerEditor], ['Description', () => Serenity.StringEditor], ['DecisionNumber', () => Serenity.IntegerEditor], ['ResponsibleContactId', () => Serenity.IntegerEditor], ['DueDate', () => Serenity.DateEditor], ['ResolutionStatus', () => Serenity.IntegerEditor], ['Images', () => Serenity.StringEditor], ['Attachments', () => Serenity.StringEditor]].forEach(x => Object.defineProperty(MeetingDecisionForm.prototype, <string>x[0], { get: function () { return this.w(x[0], (x[1] as any)()); }, enumerable: true, configurable: true }));
}

